src/default-routes.js

```diff
@@ -3,14 +3,14 @@ import { Route } from 'react-router'
 import CategoriesLoader from './loader-categories'
 import PostLoader from './loader-post'
 import PageLoader from './loader-page'
-import HomeLoader from './loader-home'
+import FrontPageLoader from './loader-frontpage'
```

Comment:
`/loader-front-page` to stick with [WordPress naming convention](https://developer.wordpress.org/themes/basics/template-hierarchy/#front-page-display)